downstream/modules/aap-hardening/ref-installing-security-updates-only.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+// downstream/assemblies/assembly-aap-security-use-cases.adoc
+
+[id="ref-installing-security-updates-only_{context}"]
+
+= Installing security updates only
+
+[role="_abstract"]
+
+For organizations with a policy requiring that all RPMs including security errata be kept up to date, the following playbook might be used in a regularly scheduled job template.
```

Comment:
```suggestion
If your organization has a policy requiring that all RPMs including security errata be kept up to date, you can use the following playbook in a regularly scheduled job template.
```
Best to avoid "might" in technical docs